### ISSUE TITLE
MAUI ios device picker lists all simulators as 'running'

### ISSUE THREAD

**Issue by rbakhshi** [microsoft/vscode-dotnettools#677]
### Describe the Issue

When listing ios simulator and devices, all simulators (regardless of state) are listed as `running` (see attached screen)

![image](https://github.com/microsoft/vscode-dotnettools/assets/4858458/1e95feac-c243-4f18-b7fe-6b553ec2753a)

### Steps To Reproduce

1. open command center in VsCode
2. Select `.NET MAUI: Pick iOS Device`
3.Notice all simulators are listed as `running`

### Expected Behavior

The state of each simulator must match the actual state. In above example on one is running.

### Environment Information

# VS Code
```
Version: 1.84.0
Commit: d037ac076cee195194f93ce6fe2bdfe2969cc82d
Date: 2023-11-01T11:29:51.160Z
Electron: 25.9.2
ElectronBuildId: 24603566
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 22.6.0
```
# OS
`macOS Version 13.5.2 (22G91)`

> VS bug [#1912902](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1912902)

**Comment by regetskcob** [comment]
Same for me, all iPadOS and iOS simulator are displayed as "running".

![Screenshot 2023-11-09 at 12 18 31](https://github.com/microsoft/vscode-dotnettools/assets/52231719/e7ffbabf-14cf-47e6-8112-5805eff10e27)

**Comment by jknaudt21** [comment]
This was fixed in the 0.8.44 release
